keyboards/iriskeyboards/info.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
This needs to be filled out. Press Ctrl+Shift+I in the Configurator to preview it.

Reply:
I have filled it out and tested it using Ctrl+Shift+I in the Configurator!